src/providers/ParticipantProject/ParticipantProjectSpeciesProvider.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { useCallback, useEffect, useState } from 'react';
+
+import { requestListParticipantProjectSpecies } from 'src/redux/features/participantProjectSpecies/participantProjectSpeciesAsyncThunks';
+import { selectParticipantProjectSpeciesListRequest } from 'src/redux/features/participantProjectSpecies/participantProjectSpeciesSelectors';
+import { useAppDispatch, useAppSelector } from 'src/redux/store';
+import { ParticipantProjectSpecies } from 'src/services/ParticipantProjectSpeciesService';
+import { Deliverable } from 'src/types/Deliverables';
+
+import { useProjectData } from '../Project/ProjectContext';
+import { ParticipantProjectSpeciesContext, ParticipantProjectSpeciesData } from './ParticipantProjectSpeciesContext';
+
+export type Props = {
+  children: React.ReactNode;
+};
+
+const ParticipantProjectSpeciesProvider = ({ children }: Props) => {
+  const dispatch = useAppDispatch();
+
+  const { projectId } = useProjectData();
+
+  const [currentParticipantProjectSpecies, setCurrentParticipantProjectSpecies] = useState<ParticipantProjectSpecies>();
+  const [currentDeliverable, setCurrentDeliverable] = useState<Deliverable>();
+  const participantProjectSpeciesApi = useAppSelector(selectParticipantProjectSpeciesListRequest(projectId));
+
+  const reload = useCallback(() => {
+    void dispatch(requestListParticipantProjectSpecies(projectId));
```

Comment:
Can you pull the current species by using the path param instead of the requesting the entire list? You can use `requestGetParticipantProjectSpecies(participantProjectSpeciesId)`